src/WebUI/src/pages/shop.vue

```diff
@@ -193,6 +193,7 @@ const newItemCount = computed(
       sticky-header
       :detailed="isUpgradableCategory"
       detailKey="id"
+      customRowKey="id"
```

Comment:
this is an important, fixes broken `Tooltip` when changing the filter by item type